### PR TITLE
This is a test to get notification on mobile

### PR DESCRIPTION
**Pull request by solarix09**
PR review to get notified on mobile

### CODE REVIEW

**Comment by solarix09** [comment]
hello, need your reply here @danicaokamoto

**Comment by danicaokamoto** [comment]
Got the notification vibration :) It works in android

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# t-e-s-t
\ No newline at end of file
+# t-e-s-t
+
+He walked on the waterrrrr
```
Comment (danicaokamoto) [inline README.md:3]: Yeah!

**Review by danicaokamoto — COMMENTED** [review]
All good

**Pull request merged**